.github/workflows/benchmark-tags.yml

```diff
@@ -0,0 +1,74 @@
+name: Benchmark Tags
+
+on:
+  workflow_dispatch:
+
+jobs:
+  sdk-benchmark:
+    name: Benchmark SDK
+    runs-on: self-hosted
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        tag-version:
+          - v1.6.0
+          - v1.7.0
+          - v1.7.1
+          - v1.10.0
+          - v1.10.1
+          - v1.11.0
+          - v1.12.0
+          - v1.13.0
+          - v1.14.0
+          - v1.15.0
+          - v1.16.0
+          - v1.17.0
+          - v1.18.0
+          - v1.19.0
+          - v1.21.0
+          - v1.22.0
+          - v1.23.0
+          - v1.23.1
+          - v1.24.0
+          - v1.25.0
+          - v1.26.0
+          - v1.27.0
+          - v1.28.0
+          - v1.29.0
+          - v1.30.0
+          - v1.30.1
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ matrix.tag-version }}
+
+      - id: setup-java
+        name: Set up Java for build
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 17
+
+      - uses: gradle/gradle-build-action@v2
+        with:
+          arguments: |
+            jmhJar
+        env:
+          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+
+      - name: Run Benchmark
+        run: |
+          cd sdk/trace/build
+          java -jar libs/opentelemetry-sdk-trace-*-jmh.jar -rf json SpanBenchmark SpanPipelineBenchmark ExporterBenchmark
```

Comment:
What's the advantage of this over `./gradlew :sdk:trace:jmh -PjmhIncludeSingleClass=SpanBenchmark`?

Reply:
I didn't try that approach. This seems equivalent to me.